### PR TITLE
Add structured encrypt example (GEA-11385)

### PR DESCRIPTION
**Pull request by kenany**

### CODE REVIEW

File: examples/vault/encrypt_structured.mjs [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+const config = new PangeaConfig({ domain: process.env.PANGEA_DOMAIN });
+const vault = new VaultService(token, config);
+
+(async () => {
+  // First create an encryption key, either from the Pangea Console or
+  // programmatically as below.
+  const createResponse = await vault.symmetricGenerate(
+    Vault.SymmetricAlgorithm.AES256_CFB,
+    Vault.KeyPurpose.ENCRYPTION,
+    "any unique name"
```
Comment (pangea-andrest) [inline examples/vault/encrypt_structured.mjs:17]: Same comment here https://github.com/pangeacyber/pangea-go/pull/158#discussion_r1449234519

File: examples/vault/package.json [hunk 2]
```diff
@@ -2,13 +2,10 @@
...
-  "main": "index.js",
-  "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
-  },
+  "private": true,
   "author": "Pangea",
   "license": "MIT",
   "dependencies": {
-    "pangea-node-sdk": "3.5.0"
+    "pangea-node-sdk": "file:../../packages/pangea-node-sdk"
```
Comment (pangea-andrest) [inline examples/vault/package.json:9]: How do you want to handle it? Next version should be 3.6.0

**Review by pangea-andrest — COMMENTED** [review]
Commented

**Review by pangea-andrest — DISMISSED** [review]

File: examples/vault/encrypt_structured.mjs (hunk 1 above)
Comment (kenany) [reply examples/vault/encrypt_structured.mjs]: Done.

File: examples/vault/package.json (hunk 2 above)
Comment (kenany) [reply examples/vault/package.json:9]: This approach will be better once coupled with CI for the examples, as it'll catch incoming changes that break examples.

**Review by pangea-andrest — APPROVED** [review]

**Pull request merged**